.github/workflows/release-drafter.yml

```diff
@@ -0,0 +1,31 @@
+name: Release drafter
+
+on:
+  push:
+    tags:
+      - "*"
```

Comment:
Can we add a comment here: `# Push events to every tag not containing /`

Reply:
Sorry I didn't quite get that. This workflow will be triggered irrespective of any type of tag. Is there a plan to push specifically named tags?
Also my understanding is tag will be pushed when you want to release this product. Correct me if I am wrong please.

Reply:
From the [documentation](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#patterns-to-match-branches-and-tags) it looks like `*` is for `Matches all branch and tag names that don't contain a slash (/).`

I just wanted to add that here as a comment.

Reply:
The `tags` options on line 5 will look only for tags and not branches. I can add a comment you asked just for as an information.

Reply:
Yes please.

Reply:
Done